clang/test/Driver/amdgpu-hip-system-arch.c

```diff
@@ -1,6 +1,4 @@
 // REQUIRES: system-linux
```

Comment:
Unrelated, but I don't see why this would fail on e.g. a FreeBSD host. Maybe it should be !WINDOWS? Although that should also be handled by REQUIRES: shell

Reply:
I hope you feel empowered to post more cleanup patches upstream!